clients/java/src/main/java/io/zeebe/client/ZeebeClientCloudBuilder.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client;
+
+public interface ZeebeClientCloudBuilder extends ZeebeClientBuilder {
+
+  /**
+   * Sets the cluster id of the Camunda Cloud cluster. This parameter is mandatory.
```

Comment:
If you really want to make this parameter mandatory at compile time, you can use intermediary inner interfaces to represent steps of the builder pattern (this pattern is described [here](https://blog.jayway.com/2012/02/07/builder-pattern-with-a-twist/) and we use this already in the zeebe client [here](https://github.com/camunda-cloud/zeebe/blob/develop/clients/java/src/main/java/io/zeebe/client/api/worker/JobWorkerBuilderStep1.java)). 
In this form `withClusterId` is part of the first interface and returns the interface of the second step, etc. Finally, the interface of the third step (likely `withClientSecret`) can return a `ZeebeClientBuilder` which then allows to build or to set other properties.

Reply:
Done